network/requirements.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Node Hardware Requirements
+---
+
+Here you can find the requirements for running the various types of nodes the operate the Stellar network. We offer some modest recommendations for each type of resource, as well as a generally applicable SKU for some popular cloud computing providers. These may require customizations or adjustments, depending on your specific use-case.
+
+| Node Type | CPU | RAM | Disk | AWS SKU | Google Cloud SKU |
+| --- | --- | --- | --- | --- | --- |
+| Core Validator Node | 8x Intel Xeon @ 3.4 GHz | 16 GB | 100 GB NVMe SSD | [c5d.2xlarge] | [n4-highcpu-8] |
```

Comment:
AWS might have updated their site since [c5d.2xlarge](https://aws.amazon.com/ec2/instance-types/c5/) is showing as 200 GB NVMe SSD. 
With BucketlistDB turned on, we had seen about 30 GB drop in storage usage. But we can continue to recommend c5d.2xlarge.